feature-group-definitions/canvas-context-lost.yml

```
@@ -1,4 +1,5 @@
 name: contextlost and contextrestored
+description: The `contextlost` event for `<canvas>` fires when the canvas backing storage is lost, while the `contextrestored` is fired when it is recreated.
```

style nit: consistency

```suggestion
description: The `contextlost` event for `<canvas>` fires when the canvas backing storage is lost, while the `contextrestored` event fires when it is recreated.
```